static/css/artist/artist_list.css

```diff
@@ -74,34 +74,29 @@
     float: left;
 }
 
-.artist-name {
-    padding-top: 5px;
-    padding-bottom: 0px;
+.artist-details {
     font-family: Dosis, sans-serif;
     background-color: #ecf0f1;
+    margin: 0px;
 }
 
-.artist-location {
+.name {
```

Comment:
`name` is a little too generic, I think. I would keep the names `artist-name`, `artist-location`, etc. but just add a new class `artist-details` like you did here.